src/components/StatTrack/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import cx from 'classnames';
+
+import styles from './styles.styl';
+
+export default function StatTrack(props) {
+  const { className, statTrack, description } = props;
+
+  let value = statTrack.$instanceData.flavorObjective.progress/statTrack.$objective.completionValue;
```

Comment:
'Tracked stat' objectives have a completion value? I havent dug too much into the data for these here, but it seems odd that you have to divide to get the result...

Reply:
yes most stats have a completion value of 1 but the crucible KDA stat has a completion value of 100 so it needs to do this to make KDA 2.06 instead of 206